independent-projects/tools/registry-client/pom.xml

```diff
@@ -45,4 +45,21 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.jboss.jandex</groupId>
+                <artifactId>jandex-maven-plugin</artifactId>
+                <version>1.0.8</version>
+                <executions>
+                    <execution>
+                        <id>make-index</id>
+                        <goals>
+                            <goal>jandex</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
```

Comment:
Is change relevant?

Reply:
Yes, that fixes the `com.fasterxml.jackson.databind.JsonMappingException: Class io.quarkus.registry.catalog.json.JsonArtifactCoordsDeserializer has no default (no arg) constructor` exception, because the JsonDeserializer in this module is never found in a Quarkus application

Reply:
OK, fine with me then